src/utils/init-data.ts

```diff
@@ -1,17 +1,7 @@
 import users from '../providers/dataProvider/defaults/users'
 import { generateSalt, encryptData } from './encryption'
```

Comment:
@TahaKhanAbdalli - I don't want to remove the concept of mock data. There are times when it could be useful to have in-browser dataset.

I just want us to have a flag (in `app.tsx`?) that indicates whether to use `localForge` or `REST` backend.